### ISSUE TITLE
Backup broken

### ISSUE THREAD

**Issue by scharc** [YunoHost/issues#1716]
Backup Script is broken: 
```yunohost backup create -o /mnt/backup/
Info: Collecting files to be backed up for multi_webapp__2...
Error: Could not back up multi_webapp__2
Traceback (most recent call last):
  File "/usr/bin/yunohost", line 72, in <module>
    parser=parser
  File "/usr/lib/moulinette/yunohost/__init__.py", line 29, in cli
    top_parser=parser
  File "/usr/lib/python2.7/dist-packages/moulinette/__init__.py", line 120, in cli
    args, output_as=output_as, timeout=timeout
  File "/usr/lib/python2.7/dist-packages/moulinette/interfaces/cli.py", line 477, in run
    ret = self.actionsmap.process(args, timeout=timeout)
  File "/usr/lib/python2.7/dist-packages/moulinette/actionsmap.py", line 592, in process
    return func(**arguments)
  File "/usr/lib/moulinette/yunohost/backup.py", line 2056, in backup_create
    backup_manager.collect_files()
  File "/usr/lib/moulinette/yunohost/backup.py", line 507, in collect_files
    self._collect_apps_files()
  File "/usr/lib/moulinette/yunohost/backup.py", line 639, in _collect_apps_files
    self._collect_app_files(app_instance_name)
  File "/usr/lib/moulinette/yunohost/backup.py", line 716, in _collect_app_files
    filesystem.rm(tmp_script, force=True)
UnboundLocalError: local variable 'tmp_script' referenced before assignment
```
```
yunohost -v
yunohost: 
  repo: stable
  version: 4.1.5
yunohost-admin: 
  repo: stable
  version: 4.1.4
moulinette: 
  repo: stable
  version: 4.1.4
ssowat: 
  repo: stable
  version: 4.1.2
```

**Comment by alexAubin** [comment]
I fixed the ugly stack_trace about tmp_script not existing ... but the fact that this happened is itself puzzling and means that yunohost was not able to create some temporary directory to put files in during the backup process ...

Can you provide more context on the exact command or procedure you used to create the backup ..

**Comment by alexAubin** [comment]
Closing due to lack of feedback, the issue with the stacktrace should be fixed in 4.1.7, would recommend to use the custom webapp / my_webapp instead of "multi_webapp" if the issue is app-specific
